### PR TITLE
Update Portuguese translations

### LINKED ISSUE #5961: Zhongwen & Portuguese(Brasil) review
**Is your feature request related to a problem? Please describe.**
We want to release Zhongwen and Portuguese front pages. We should check all translations, configuration, and links. 

**Describe the solution you'd like**
- We should check requirements with Jon.
- We should check the service config as well as markup of the page to ensure we have translations for all offscreen text as well as on screen text.
- We should check Chartbeat config, ATI config, consent banners, metadata, navigation links, footer links, trust project links. Note that this should be done for these pages:

https://www.test.bbc.com/zhongwen/simp
https://www.test.bbc.com/zhongwen/trad
https://www.test.bbc.com/portuguese

- If there are any bugs, they should be fixed.
- Then we should review with UX designers (Laura and Ed).

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Unit tests should be sufficient, plus manual checks.

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #5961

**Overall change:** 
Updated Portuguese translations with these findings https://github.com/bbc/simorgh/issues/5961#issuecomment-604330449 .

**Code changes:**
- Updated Portuguese config file.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/portuguese.js [hunk 1]
```diff
@@ -205,12 +205,6 @@ export const service = {
           href: 'https://www.bbc.com/portuguese/institutional-36202456',
           text: 'Fale com a BBC',
         },
-        {
-          href:
-            'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
-          text: 'AdChoices / Do Not Sell My Info',
```
Comment (sareh) [inline src/app/lib/config/services/portuguese.js]: We need to have this additional footer link for all services. Unfortunately the reference spreadsheet hasn't been updated to reflect this. We should ask Jon to update that.

Comment (OlgaLyubin) [reply src/app/lib/config/services/portuguese.js]: Thanks 👍 Added it back

File: src/app/lib/config/services/portuguese.js [hunk 2]
```diff
@@ -177,7 +177,7 @@ export const service = {
     },
     footer: {
```
Comment (sareh) [inline src/app/lib/config/services/portuguese.js:180]: `noBylinesPolicy` should be updated to this anchor in the trust project page: `https://www.bbc.com/portuguese/institutional-50054434#authorexpertise`
`publishingPrinciples` should be updated to the trust project page: `https://www.bbc.com/portuguese/institutional-50054434`

Comment (sareh) [reply src/app/lib/config/services/portuguese.js:180]: I just realised that this was also missed when we reviewed the Zhongwen config updates. https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/zhongwen.js#L266-L267 Could you please update this as well in this PR, so we don't have to open a new PR? Thank you.

Comment (OlgaLyubin) [reply src/app/lib/config/services/portuguese.js:180]: These are updated now

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by Bopchy — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
This doesn't need manual testing, so moving it to ready to merge..

**Pull request merged**